website/client/components/chat/chatMessages.vue

```diff
@@ -145,7 +154,29 @@ export default {
       return this.chat;
     },
   },
+  watch: {
+    messages () {
+      // @TODO: MAybe we should watch insert and remove?
+      this.loadProfileCache();
+    },
+  },
   methods: {
+    async loadProfileCache () {
+      let promises = [];
+
+      this.messages.forEach(message => {
+        let uuid = message.uuid;
+        if (!this.cachedProfileData[uuid]) {
+          promises.push(axios.get(`/api/v3/members/${uuid}`));
+        }
+      });
+
+      let results = await Bluebird.all(promises);
```

Comment:
No difference but I think it's the first time I've seen bluebird used in the client so if we avoid loading it is one less dependency